.github/workflows/continous-integration.yml

```
@@ -1148,12 +1165,35 @@ jobs:
 
       - name: Build ⚒️ Distributions
         run: poetry build
+      
+      # TODO: readd release step to puplic pypi
```

just a tiny suggestion to make this process easier :)
```suggestion
      # TODO: readd release step to puplic pypi
      # - name: Publish to PyPI 📦
      #    uses: pypa/gh-action-pypi-publish@717ba43cfbb0387f6ce311b169a825772f54d295
      #    with:
      #      user: __token__
      #      password: ${{ secrets.PYPI_TOKEN }}
```